src/edu/ucsb/cs56/projects/games/states_quiz/GamePanel.java

```diff
@@ -68,12 +70,18 @@ public GamePanel(Runnable reloadFrame) {
         this.add(mapPanel, BorderLayout.CENTER);
 
         this.add(questionScrollPane, BorderLayout.SOUTH);
-        this.add(answerScrollPane, BorderLayout.EAST);
+	// this.add(answerScrollPane, BorderLayout.EAST);
```

Comment:
If you're "turning off" code you might as well delete it.  You can get it back from your Git history if it turns out you actually needed it.